auction-app-frontend/src/pages/ProductDetails/ProductDetails.jsx

```diff
@@ -0,0 +1,106 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+import { Tabs, LoadingComponent, ErrorComponent } from "src/components";
+import { useBreadcrumb } from "src/store/BreadcrumbContext";
+
+import { getProduct, getProductImages } from "src/services";
+
+import { PRODUCT_DETAILS_TABS } from "src/constants";
+
+import "./style.scss";
+
+const ProductDetails = () => {
+  const [activeTab, setActiveTab] = useState(PRODUCT_DETAILS_TABS[0].id);
+  const [product, setProduct] = useState(null);
+  const [productImages, setProductImages] = useState([]);
+  const [mainImage, setMainImage] = useState("");
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+  const { id } = useParams();
+  const { setTitle } = useBreadcrumb();
+
+  const fetchInitialData = () => {
+    setLoading(true);
+
+    Promise.all([getProduct(id), getProductImages(id)])
```

Comment:
Update this to match new approach with images being part of the product and in other places also.